### PR TITLE
Remove unused message types

### PR DESCRIPTION
**Pull request by jzzheng22**
# Summary

### CODE REVIEW

**Review by hussain2603 — DISMISSED** [review]

**Comment by JaafarRammal** [comment]
Hmmm I think this breaks the messages logger which assumes 14 message types

File: pkg/messages/baseMessage.go [hunk 1]
```diff
@@ -7,15 +7,12 @@ import "github.com/google/uuid"
...
 	AskFoodTaken MessageType = iota + 1
 	AskHP
-	AskFoodOnPlatform
 	AskIntendedFoodIntake
 	StateFoodTaken
 	StateHP
-	StateFoodOnPlatform
 	StateIntendedFoodIntake
 	StateResponse
 	ProposeTreaty
```
Comment (JaafarRammal) [inline pkg/messages/baseMessage.go:18]: Hmmm I think this breaks the messages logger which assumes 14 message types (bad code design sorry)

Comment (jzzheng22) [reply pkg/messages/baseMessage.go:18]: If it's too annoying to fix then let's just close this PR

Comment (JaafarRammal) [reply pkg/messages/baseMessage.go:18]: just realized we need it. if message graphs are fine with now warnings let's merge it

**Review by JaafarRammal — APPROVED** [review]

**Review by JaafarRammal — APPROVED** [review]
Tested, LGTM

**Pull request merged**